.github/workflows/integration.yml

```diff
@@ -3,6 +3,7 @@ name: "INTEGRATION"
 env:
   HARDHAT_VAR_MNEMONIC: "test test test test test test test test test test test junk"
   HARDHAT_VAR_INFURA_API_KEY: "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+  PRIVATE_KEY: "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
Not necessary right now, but we should move all of these to environmental variables at some point, just so we or others don't accidentally put sensitive information here.

Reply:
Agreed, this PK specifically is for the integration tests to pass as it needs to publish the PubKey and Decrypted PT